xclim/indices/_threshold.py

```diff
@@ -1296,12 +1296,17 @@ def heating_degree_days(
 
     Notes
     -----
+    This index intentionally differs from it's ECA&D equivalent: HD17.
+    In HD17, values below zero are not clipped before the sum.
+    The present definition should gives a better representation of the energy demand
+    for cooling building to the given threshold.
```

Comment:
```suggestion
    This index intentionally differs from its ECA&D equivalent: HD17. In HD17, values below zero are 
    not clipped before the sum. The present definition should provide a better representation of the energy
    demand for cooling buildings to the given threshold.
```

Reply:
I think I never get the documentation right, what are the rules to follow here ?

Reply:
The docstring was more or less fine! There were some grammar issues that I adjusted. In terms of spacing, there's no need to separate sentences by lines, as ReStructuredText will group it all together anyway (a new line in a paragraph counts as a space when the documentation is generated). So long as the line length remains less than ~90 characters long (including preceding spaces), all is good!

Reply:
Do you think this information would be beneficial in the contributors docs?

Reply:
Well I checked in contributors doc before asking my question so I guess it would make sense. 
But it would be sufficient to link to an existing resource of "how to write rst", unless Xclim have some specific rules overriding these.
In my case I haven't read enough about rst format so it would be relevant.